threads/tasks.py

```diff
@@ -52,3 +60,53 @@ def archive_thread(payload):
     thread_found.archived = True
     thread_found.save()
     return thread_found
+
+
+def unarchive_thread(payload):
+    thread_found = Thread.objects.get(pk=payload["thread_id"])
+    thread_found.archived = False
+    thread_found.save()
+    return thread_found
+
+
+def edit_comment(payload):
+    comment_found = Comment.objects.get(pk=payload["comment_id"])
+    comment_found.content = payload["content"]
+    comment_found.updated_at = payload["created_at"]
+    comment_found.thread.updated_at = payload["created_at"]
+    comment_found.save()
+    return comment_found
+
+
+def trash_thread(payload):
+    thread_found = Thread.objects.get(pk=payload["thread_id"])
+    thread_found.trashed = True
+    thread_found.save()
+    return thread_found
+
+
+def restore_thread(payload):
+    thread_found = Thread.objects.get(pk=payload["thread_id"])
+    thread_found.trashed = False
+    thread_found.save()
+    return thread_found
+
+
+def trash_comment(payload):
+    comment_found = Comment.objects.get(pk=payload["comment_id"])
+    comment_found.trashed = True
+    comment_found.save()
+    return comment_found
+
+
+def restore_comment(payload):
+    comment_found = Comment.objects.get(pk=payload["comment_id"])
+    comment_found.trashed = False
+    comment_found.save()
+    return comment_found
+
+
+def replace_user(payload):
```

Comment:
This isn't correct yet. The payload you receive here should be similar to the one in `create_thread`, i.e. you receive both `provider_id` and `user_id` for the old and new user. And you have to possibly create the user